src/Conditionals.elm

```diff
@@ -0,0 +1,489 @@
+trafficLightActionIfElse color =
+    if color == "Red" then
+        "Stop"
+        -- This line checks if input color is "Red" then
+        -- return "Stop"
+
+    else if color == "Yellow" then
+        "Prepare to stop"
+        -- This line checks if input color is "Yellow" then
+        -- return "Prepare to stop"
+
+    else if color == "Green" then
+        "Go"
+        -- This line checks if input color is "Green" then
+        -- return "Go"
+
+    else
+        "Are we on Mars?"
+
+
+
+-- Anything else outside of these colors ("Red", "Yellow", and "Green")
+-- then it will return "Are we on Mars?"
+
+
+trafficLightActionCase : String -> String
+trafficLightActionCase color =
+    case color of
+        "Red" ->
+            "Stop"
+
+        "Yellow" ->
+            "Prepare to stop"
+
+        "Green" ->
+            "Go"
+
+        _ ->
+            color ++ "? Are we on Mars?"
+
+
+convertTo12HourFormat : Int -> ( Int, String )
+convertTo12HourFormat hour =
+    if hour >= 0 && hour <= 23 then
+        let
+            ( convertedHour, period ) =
+                if hour < 12 then
+                    if hour == 0 then
+                        ( 12, "AM" )
+
+                    else
+                        ( hour, "AM" )
+
+                else if hour == 12 then
+                    ( 12, "PM" )
+
+                else
+                    ( hour - 12, "PM" )
+        in
+        ( convertedHour, period )
+
+    else
+        ( -1, "Extra time? This isn't football, hun!" )
+
+
+weekdayMoodIfElse : String -> String
+weekdayMoodIfElse day =
+    if day == "Monday" then
+        "Back to the grind!"
+
+    else if day == "Tuesday" || day == "Wednesday" || day == "Thursday" then
+        "Just another day in the grind"
+
+    else if day == "Friday" then
+        "TGIF!"
+
+    else if day == "Saturday" || day == "Sunday" then
+        "It's the weekend, baby!"
+
+    else
+        "Is that a public holiday?"
+
+
+weeklydayMoodCase : String -> String
+weeklydayMoodCase day =
+    case day of
+        "Monday" ->
+            "Back to the grind!"
+
+        "Tuesday" ->
+            "Just another day in the grind"
+
+        "Wednesday" ->
+            "Just another day in the grind"
+
+        "Thursday" ->
+            "Just another day in the grind"
+
+        "Friday" ->
+            "TGIF!"
+
+        "Saturday" ->
+            "It's the weekend, baby!"
+
+        "Sunday" ->
+            "It's the weekend, baby!"
+
+        _ ->
+            "Is that a public holiday?"
+
+
+fizzBuzzGame : Int -> String
+fizzBuzzGame number =
+    if modBy 3 number == 0 && modBy 5 number == 0 then
+        "FizzBuzz"
+
+    else if modBy 3 number == 0 then
+        "Fizz"
+
+    else if modBy 5 number == 0 then
+        "Buzz"
+
+    else if number < 0 then
+        "Buzzing into the negatives, are we?"
+
+    else if number > 100 then
+        String.fromInt number ++ " - Just another number in the universe!"
```

Comment:
We should probably check these first, as they catch numbers we don't want to fizzbuzz for.

Right now, if you pass in 110, it prints `Buzz`. Because it gets caught by the buzz branch. Please try it out by hand (without using your computer or repl) and see that it gets caught by that branch.